docs/plugins/events.md

```diff
@@ -0,0 +1,45 @@
+# Events
+
+Generic Events emitter / listener interface which can be used for _any_ type of event. For Lifecycle events please see [Lifeycle](/plugins/lifecycle.md)
```

Comment:
Should we allow developers to mix this in to their own objects?
```js
import { Events } from '@lightningjs/sdk'
const object = {}
const handler = (_args) => console.log('initialized')
object.mixin(Events)

object.addEventListener('initialize', handler)
```

Reply:
hey @cletustboone this looks like an interesting concept, but I'm not sure yet what problem it solves? Can you give an example of what `object` would be in a real App (as it's just an empty object literal now :) )

Reply:
I was thinking about a component being able to do something with itself when something changes.
```js
someComponent.addEventListener('change:field', function() {someComponent.render()})
```
But then I remembered, there's already some existing mechanism in Lightning to accomplish this.